behat.yml.dist

```diff
@@ -0,0 +1,23 @@
+default:
+  suites:
+    default:
+      paths:
+        - %paths.base%/tests/features
+      contexts:
+        - Drupal\DrupalExtension\Context\MinkContext
+        - Drupal\DrupalExtension\Context\DrupalContext
+        - Drupal\DrupalExtension\Context\MarkupContext
+        - Drupal\Tests\oe_webtools\Behat\DrupalContext
```

Comment:
- Drupal\DrupalExtension\Context\MarkupContext
 - Drupal\Tests\oe_webtools\Behat\DrupalContext
These classes don't seem to exist